src/plugins/intel_cpu/src/emitters/plugin/riscv64/jit_eltwise_emitters.hpp

```diff
@@ -181,6 +181,27 @@ class jit_greater_equal_emitter : public jit_emitter {
     void emit_isa(const std::vector<size_t>& in_vec_idxs, const std::vector<size_t>& out_vec_idxs) const;
     void register_table_entries() override;
 };
+
+class jit_less_equal_emitter : public jit_emitter{
+public:Add commentMore actions
```

Comment:
Looks like you accidentally left this comment here - because of that the [builds ](https://github.com/openvinotoolkit/openvino/actions/runs/15775525351/job/44480802127?pr=31058)are failed. Please remove it